packages/common/src/types/auth.ts

```diff
@@ -56,6 +57,16 @@ export type OauthAuth = {
     resource?: URL;
 };
 
+export type OssEmbed = {
```

Comment:
will this be backwards compatible ? as in,  can an existing token have access to the fucntionality, without throwing errors about a missing projectuuid ?

Reply:
Naming is hard. OssEmbed is taken from the Embed DB type where `projectUuid` is a required. I just copied it out of the EE folder so that we could use the type in OSS. It's then aliased as Embed so as not to need to change many more files that are unrelated to the scope of this PR. 

https://github.com/lightdash/lightdash/blob/5fd12eed8d9b4b784257586697ade51d03b9af1e/packages/common/src/ee/embed/index.ts#L6-L14

The JWT type has the optional `projectUuid` that I missed as being required:
https://github.com/lightdash/lightdash/blob/5fd12eed8d9b4b784257586697ade51d03b9af1e/packages/common/src/ee/embed/index.ts#L103-L116